skeleton/template.yaml

```diff
@@ -80,12 +80,36 @@ spec:
                   title: Model Server Endpoint
                   type: string
                   description: "The endpoint for an existing model server."
+                includeModelEndpointSecret:
+                  title: Bearer Authentication Required?
+                  type: boolean
+                  default: false
                 # SED_LLM_SERVER_START
                 modelName:
                   title: Model Name
                   type: string
                   ui:help: "The name of the model deployed on the model server you would like to use."
                 # SED_LLM_SERVER_END
+              dependencies:
+                includeModelEndpointSecret:
+                  allOf:
+                    - if:
+                        properties:
+                          includeModelEndpointSecret:
+                            const: true
+                      then:
+                        properties:
+                          modelEndpointSecretName:
+                            title: Model Enpoint Secret Name
+                            ui:help: Please provide the name of the secret that your bearer token is stored.
+                            type: string
+                          modelEndpointSecretKey:
+                            title: Model Enpoint Secret Key
+                            ui:help: Please provide the key of the secret that the token value is assigned to.
```

Comment:
```suggestion
                            ui:help: Provide the Secret's key containing the token value.
```

Reply:
IMO this should probably be `Provide the key of the Secret containing the token value.`